typedjsonrpc/registry.py

```diff
@@ -124,6 +134,8 @@ def _handle_exceptions(self, method, is_notification=False, msg_id=None):
                     debug_url = self._store_traceback()
                 else:
                     debug_url = None
+                exception_message = "id: {}, debug_url: {}".format(msg_id, debug_url)
+                self._logger.exception(exception_message)
```

Comment:
`self._logger.exception` takes in an exception. We should be using `self._logger.error` instead